Gems/MiniAudio/Code/Source/Tools/EditorMiniAudioListenerComponent.cpp

```diff
@@ -55,6 +55,11 @@ namespace MiniAudio
 
                     ->DataElement(AZ::Edit::UIHandlers::Default, &MiniAudioListenerComponentConfig::m_followEntity, "Follow Entity", "The listener will follow the position and orientation of the specified entity.")
                     ->DataElement(AZ::Edit::UIHandlers::Default, &MiniAudioListenerComponentConfig::m_listenerIndex, "Listener Index", "MiniAudio listener index to control.")
+                    ->DataElement(AZ::Edit::UIHandlers::Default, &MiniAudioListenerComponentConfig::m_globalVolume, "Global Volume", "Sets the global volume of the audio engine, as a percentage.")
```

Comment:
Might be worth switching it to UIHandlers::Slider here, instead of Default, since its a pretty well bounded value?  Super easy way to make it easier to use.

Reply:
Didn't see you made the same comment :)

Reply:
I implemented this in 3128639513ef631f8493e388f752be9999ed7b01